src/gmp/commands/reports.ts

```diff
@@ -0,0 +1,231 @@
+/* SPDX-FileCopyrightText: 2024 Greenbone AG
+ *
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+import {convertBoolean} from 'gmp/commands/convert';
+import EntitiesCommand from 'gmp/commands/entities';
+import EntityCommand from 'gmp/commands/entity';
+import GmpHttp from 'gmp/http/gmp';
+import DefaultTransform from 'gmp/http/transform/default';
+import {XmlResponseData} from 'gmp/http/transform/fastxml';
+import logger from 'gmp/log';
+import Filter, {ALL_FILTER} from 'gmp/models/filter';
+import Report, {ReportElement} from 'gmp/models/report';
+import {isDefined} from 'gmp/utils/identity';
+
+interface ReportCommandImportParams {
+  task_id: string;
+  in_assets?: number;
+  xml_file: string;
```

Comment:
For now while we don't have form validation in the frontend I would make it optional. When calling the method, undefined is allowed and only the backend checks if it is required.
```suggestion
  xml_file?: string;
```